scripts/linux.sh

```diff
@@ -22,27 +25,27 @@ for disk in $DISKS; do
   fi
 
   device="/dev/$disk"
-  diskinfo=($(lsblk -b -d $device --output SIZE,RO,RM,MODEL | ignore_first_line))
+  diskinfo=($(lsblk -b -d "$device" --output SIZE,RO,RM,MODEL | ignore_first_line))
   size=${diskinfo[0]}
   protected=${diskinfo[1]}
   removable=${diskinfo[2]}
-  description=${diskinfo[@]:3}
-  mountpoints=`get_mountpoint $device`
+  description=${diskinfo[*]:3}
+  mountpoints="$(get_mountpoint "$device")"
 
   # If we couldn't get the mount points as `/dev/$disk`,
   # get the disk UUIDs, and check as `/dev/disk/by-uuid/$uuid`
   if [ -z "$mountpoints" ]; then
-    for uuid in `get_uuids $device`; do
-      mountpoints=$mountpoints`get_mountpoint /dev/disk/by-uuid/$uuid`
+    for uuid in $(get_uuids "$device"); do
+      mountpoints="$mountpoints$(get_mountpoint "/dev/disk/by-uuid/$uuid")"
     done
   fi
 
   # If we couldn't get the description from `lsblk`, see if we can get it
   # from sysfs (e.g. PCI-connected SD cards that appear as `/dev/mmcblk0`)
   if [ -z "$description" ]; then
-    subdevice=`echo "$device" | cut -d'/' -f3`
+    subdevice="$(echo "$device" | cut -d '/' -f3)"
```

Comment:
If you're adding a space after `-d`, should there also be a space after `-f` ? ;)